cli/azd/pkg/exec/util_test.go

```diff
@@ -49,14 +49,27 @@ func TestKillCommand(t *testing.T) {
 	s := time.Now()
 
 	runner := NewCommandRunner(nil)
-	_, err := runner.Run(ctx, RunArgs{
-		Cmd: "pwsh",
-		Args: []string{
-			"-c",
-			"sleep",
-			"10000",
-		},
-	})
+	var args RunArgs
+	if runtime.GOOS == "windows" {
+		args = RunArgs{
+			Cmd: "pwsh",
+			Args: []string{
+				"-c",
+				"sleep",
+				"10000",
+			},
+		}
+	} else {
+		args = RunArgs{
+			Cmd: "sh",
+			Args: []string{
+				"-c",
+				"sleep 10s",
```

Comment:
```suggestion
				"sleep 10",
```


FWIW - I am not sure if this works on macOS:

It seems like having the `s` suffix does not work well on macOS.  I validated that without the `s` suffix `sleep 10` sleeps for 10 seconds on linux as well (using `ubuntu:latest`)

```
matell@Matts-Work-MacBook ~ % sh -c "sleep 10s"
usage: sleep seconds
```

Reply:
I am guessing the CI test passes because the command just ends up terminating immediately instead of sleeping?